fontbe/src/features.rs

```diff
@@ -56,6 +58,8 @@ pub use marks::create_mark_work;
 
 const DFLT_SCRIPT: Tag = Tag::new(b"DFLT");
 const DFLT_LANG: Tag = Tag::new(b"dflt");
+// https://learn.microsoft.com/en-us/typography/opentype/spec/name#name-ids
+const FIRST_NON_RESERVED_NAME_ID: u16 = 256;
```

Comment:
I think I've seen this twice, I suggest putting it in fontdrasil which both fea-rs and fontbe are entitled to reference